pkg/sql/virtual_schema.go

```diff
@@ -194,25 +194,8 @@ func (t virtualSchemaTable) initVirtualTableDesc(
 		if index.NumColumns() > 1 {
 			panic("we don't know how to deal with virtual composite indexes yet")
 		}
-		idx := *index.IndexDesc()
-		// All indexes of virtual tables automatically STORE all other columns in
-		// the table.
-		idx.StoreColumnIDs = make([]descpb.ColumnID, len(mutDesc.Columns)-len(idx.ColumnIDs))
-		idx.StoreColumnNames = make([]string, len(mutDesc.Columns)-len(idx.ColumnIDs))
-		// Store all columns but the ones in the index.
-		outputIdx := 0
-	EACHCOLUMN:
-		for j := range mutDesc.Columns {
-			for _, id := range idx.ColumnIDs {
-				if mutDesc.Columns[j].ID == id {
-					// Skip columns in the index.
-					continue EACHCOLUMN
-				}
-			}
-			idx.StoreColumnIDs[outputIdx] = mutDesc.Columns[j].ID
-			idx.StoreColumnNames[outputIdx] = mutDesc.Columns[j].Name
-			outputIdx++
-		}
```

Comment:
Similarly as in https://github.com/cockroachdb/cockroach/pull/65133/files#r632197699 I believe this should be excluding virtual columns.